adam-core/src/main/scala/org/bdgenomics/adam/converters/AlignmentRecordConverter.scala

```diff
@@ -21,7 +21,12 @@ import htsjdk.samtools.{ SAMFileHeader, SAMRecord }
 import org.bdgenomics.adam.instrumentation.Timers._
 import org.bdgenomics.adam.models._
 import org.bdgenomics.adam.rich.RichAlignmentRecord
-import org.bdgenomics.formats.avro.AlignmentRecord
+import org.bdgenomics.formats.avro.{
```

Comment:
These imports can all be on a single line, e.g. `import org.bdgenomics.formats.avro.{AlignmentRecord, Fragment, Sequence}`